public/bundle.js

```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Looks like you may have not put your bundle.js in your GitIgnore.

Reply:
Hmmm, strange. I'll see what's up here... It's listed as "public/bundle.js" in .gitignore, but this obviously didn't work. Do you have any thoughts as to why? No worries if not -- I can research.

Reply:
I figured it out -- the file had been tracked prior to my adding it to .gitignore, so I had to manually remove it via git rm --cached public/bundle.js